alerts/hbase/README.md

```diff
@@ -0,0 +1,48 @@
+# Alert Policies for Apache HBase in the Ops Agent
+
+## Slow Operations
+
+Slow operations can be an indication that the HDFS may be in an unhealthy state.
+
+## HBase Server Stopped
+
+If the HBase has unexpected downtime, this alert will help teams discern that the server is stopped and unable to function in their environment.
+
+Required Log Query:
+
+```sh
+logName="projects/<project_id>/logs/hbase_system"
+jsonPayload.message="regionserver.HeapMemoryManager: Stopping"
+```
+
+Replace <project_id> with your Project ID
+
+## Authentication Errors
+
+Authentication errors could indicate that clients do not have correct credentials or somebody is maliciously trying to access the HBase system.
```

Comment:
Will this trigger if 1 authentication error is present (i.e. through a Log Based Alert) or after a certain number of errors are present (i.e. through a metric threshold alert?) Could we please add the trigger information to the description? Thanks!

Reply:
And could the updates that are made in the ReadMe also be passed down to the alert policy JSON below. Thank you